assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,144 @@
-console.log('***** Music Collection *****')
+console.log('***** Music Collection *****');
+
+// creating empty collection
+let collection = [];
+
+// start addToCollection 
+function addToCollection(title, artist, yearPublished, tracks) {
+
+    // create album object
+    var album = {
```

Comment:
What's this `var` doing here?

There is pretty much no reason to ever use `var` in modern JavaScript. `let` and `const` can work in every use case you might have for `var` and behave more predictably.

Reply:
honestly - no idea lol, I don't think I've ever used it outside of this instance? definitely noted and will lose whatever urge made me write var lol